pkg/apis/options/secret_source.go

```diff
@@ -0,0 +1,14 @@
+package options
+
+// SecretSource references an individual secret value.
+// Only one source within the struct should be defined at any time.
+type SecretSource struct {
+	// Value expects a base64 encoded string value.
+	Value string `json:"value,omitempty"`
+
+	// FromEnv expects the name of an environment variable.
+	FromEnv string `json:"fromEnv,omitempty"`
```

Comment:
Do we still need `fromEnv` now that we have env var substitution?

Reply:
Not really. I just wanted to keep as much backwards compatibility as possible